app.js

```diff
@@ -1,12 +1,24 @@
 const express = require('express');
 const app = express();
-const { getCategories } = require("./controller.js")
+const { getCategories, getReviewID } = require("./controller.js")
 
+app.use(express.json());
 
 app.get("/api/categories", getCategories)
 
+app.get("/api/reviews/:review_id", getReviewID)
+
+
 app.use((err, req, res, next) => {
-    res.status(500).send("Server Error!")
+    if (err.status && err.msg) {
+        res.status(404).send({ msg: err.msg })
+    } else if (err.code === "22P02") {
+        res.status(400).send({ msg: "Not a valid user ID" })

```

Comment:
May cause issues later on, fine for now but just a heads up

Reply:
^ to clarify, this is because this code could happen in any table, not just user ids